public/index.html

```diff
@@ -25,6 +25,8 @@ <h5>Color:</h5>
       <input type="radio" name="color" value="#ffbf00"> Yellow <br>
       <input type="radio" name="color" value="#FF8C00"> Orange <br>
       <input type="radio" name="color" value="red"> Red <br>
+      <input type="radio" name="color" value="Black">Black<br>
+      <input type="radio" name="color" value="#808000">Olive<br>
```

Comment:
We can take black as an options, since it is used for moderators in codeforces. But I have no idea where olive color is used. If it isn't used in any rating system, then I would suggest you to remove it.

Reply:
ok